packages/@vuepress/theme-default/src/client/styles/code-group.scss

```diff
@@ -34,6 +34,10 @@
   outline: none;
 }
 
+.code-group__nav-tab:focus-visible {
+  outline: 1px solid rgba(255, 255, 255, 0.9);
```

Comment:
Is this possible to re-use some existing color variables?

Reply:
Since it is in a dark background, I just copy the color from L29.

And other outline is ok to follow default value, so I don't think any need to add a variable